mobile/lib/widgets/asset_viewer/description_input.dart

```diff
@@ -34,17 +34,24 @@ class DescriptionInput extends HookConsumerWidget {
     final owner = ref.watch(currentUserProvider);
     final hasError = useState(false);
     final assetWithExif = ref.watch(assetDetailProvider(asset));
+    final hasDescription = useState(false);
+    final isOwner = fastHash(owner?.id ?? '') == asset.ownerId;
 
     useEffect(
       () {
-        assetService
-            .getDescription(asset)
-            .then((value) => controller.text = value);
+        assetService.getDescription(asset).then((value) {
+          controller.text = value;
+          assetHasDescription.value = value.isNotEmpty;
+        });
         return null;
       },
       [assetWithExif.value],
     );
 
+    if (!isAssetOwnedByUser && !assetHasDescription.value) {
```

Comment:
You still need to change these variable names 😄

Reply:
Really sorry about the lousiness. 😅